src/mam/mod_mam_riak_timed_arch_yz.erl

```diff
@@ -427,6 +470,8 @@ search_text_filter(SearchText) ->
 
 jid_filters(LocalJid, undefined) ->
     <<"_yz_rk:", LocalJid/binary, "*">>;
+jid_filters(undefined, RemoteJid) ->
```

Comment:
Should be a comment about that the index should be used only by gdbr code.
Otherwise it can be used accidentally by performance sensitive code.